### ISSUE TITLE
Clusters Appearing for Small Groupings, No Way to Actually Select Caches

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#4968]
**[Original report](https://bitbucket.org/zedsaidstudio/public-issues-tracker/issues/108) by Carson Monroe (Bitbucket: [ACME_WildCachers](https://bitbucket.org/ACME_WildCachers), ).**

----------------------------------------

I'm using the default settings for clustering which should mean that no cluster should represent fewer than 5 caches. However, I'm seeing clusters marked with 2 and 3 that never clear to show the actual caches even at max zoom. Comparing my offline list to the live map, these clusters are shown in the position where there's just a single cache. But because it's a cluster, you can't pull up the cache details.

These same clusters are still appearing on the map even after setting the Cluster Threshold to a value higher than the total number of caches on the list and reloading the list.

**Comment by cachly** [comment]
**Original comment by Carson Monroe (Bitbucket: [ACME_WildCachers](https://bitbucket.org/ACME_WildCachers), ).**

----------------------------------------

I played with this a bit more, thinking this might have been a corruption problem introduced when upgrading from 2.0. That does not appear to be the case. I ran a fresh version of the same pocket query used to create the offline list and it still shows the same problem on the same caches.
